app/assets/stylesheets/custom.scss

```
@@ -0,0 +1,33 @@
+@import 'bootstrap-sprockets';
+@import 'bootstrap';
+
+
+body {
+  padding-top: 60px;
+  }
```

```suggestion
body {
  padding-top: 60px;
}
```
self check indent (thụt lề) của file này nhé